sdk/synapse/synapse-artifacts/src/operationsInterfaces/bigDataPools.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import * as coreHttp from "@azure/core-http";
+import { BigDataPoolsListResponse, BigDataPoolsGetResponse } from "../models";
+
+/** Interface representing a BigDataPools. */
+export interface BigDataPools {
```

Comment:
Oh cool, you're essentially generating interfaces for clients now? That is new right?!

Reply:
Yup @sarangan12 took on this change! We no longer need to expose the operation classes 😄